bot.go

```diff
@@ -59,7 +59,12 @@ func archiveChannels(api *slack.Slack, c []slack.Channel, reason string) {
 		go func(c slack.Channel) {
 			defer wg.Done()
 			if err := api.ArchiveChannel(c.Id); err != nil {
-				log.Printf("Error archiving #%s (%s): %s\n", c.Name, c.Id, err)
+				message := fmt.Sprintf("Error archiving channel #%s (%s): %s\n", c.Name, c.Id, err)
```

Comment:
I appreciate that we're quite nested here, but is there a nice way to split this line to keep within the line length?